### ISSUE TITLE
Disable Restart when using the menu Shortcut

### ISSUE THREAD

**Issue by miotislucifugis** [probonopd/MiniDexed#134]
First, great work everyone!  Im eagerly following the progress!

one suggestion:   A few time i have accidentally triggered a reset while i was using the menu shortcut (pressing the encoder button  and scroll)  to compare settings in the TGs.   It seems to take about 10 seconds to initiate the restart, which is generous... but with this wealth of TGs, one can easily loose track of the seconds when setting parameters.        A more preferable behavior would be to disable or restart the restart timer when the encoder is being turned, to avoid any "surprises"    
Basically:
 Button held + Encoder= menu Shortcut             Button Held without encoder= start Restart countdown

**Comment by probonopd** [comment]
Happened to me once, too. If I understand @rsta2 correctly, then the functionality you describe is not that easy to implement. Or do you think you could implement it?

**Comment by miotislucifugis** [comment]
Sorry, I havent even looked at the code, im sure it is over my head anyway.  If it is difficult to change, no worries.  It is definitely not a make or break issue
best

**Comment by rsta2** [comment]
Perhaps this [reboot code](https://github.com/probonopd/MiniDexed/blob/995b25c71a4ef2a5fb8c98e77f00d2088dea36f7/src/userinterface.cpp#L212) should be totally removed? It was intended for development purposes, but 10 seconds is already very long, so switching it off and on again seems to be already quicker now.

**Comment by probonopd** [comment]
Well, maybe it's useful as a last resort in some strange circumstances. Setting the timeout to 120 seconds for now.
